website_sale_show_company_data/readme/CONTRIBUTORS.rst

```diff
@@ -1 +1,3 @@
-* David Vidal <david.vidal@tecnativa.com>
+* `Tecnativa <https://www.tecnativa.com>`_:
+  * David Vidal
```

Comment:
You need a blank line in between or it won't be properly formated (you can make a quick test with http://rst.ninjs.org)